packages/vats/src/repl.js

```diff
@@ -17,6 +32,13 @@ export function stringify(
     if (typeof value === 'bigint') {
       return `${value}n`;
     }
+    const rawString = UNJSONABLES.get(value);
+    if (rawString) {
+      return rawString;
+    }
+    if (typeof value === 'symbol') {
+      return `Symbol(?)`;
```

Comment:
Symbols carry a description string. Registered symbols have their registration string (which is probably also their description string). So you can give a lot more info that `?`. How about `String(value)` ?

Reply:
TIL:
```js
String(someSymbol) !== `${someSymbol}`
```

Thanks.  I've updated that now.